app/config/parameters.yml.dist

```diff
@@ -36,9 +36,6 @@ parameters:
     sylius.currency_importer.open_exchange_rates: EDITME
     sylius.currency_importer.ecb.base_currency: EUR
 
-    paypal.express_checkout.username: EDITME
-    paypal.express_checkout.password: EDITME
-    paypal.express_checkout.signature: EDITME
     paypal.express_checkout.sandbox: true
```

Comment:
Why leaving this one? In separate PR I would like to remove all default payment configurations from Sylius and add Cookbook articles about how to configure Stripe, PayPal, etc.